getting-started/authenticating-users/app.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The authenticating-users program is a sample web server application that
+// extracts and verifies user identity data passed to it via Identity-Aware
+// Proxy.
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+    "net/http"
+	"os"
+
+	"cloud.google.com/go/compute/metadata"
+	"github.com/dgrijalva/jwt-go"
+)
+
+var cachedCertificates = make(map[string]string)
+
+func main() {
+	http.HandleFunc("/", indexHandler)
+
+	port := os.Getenv("PORT")
+	if port == "" {
+		port = "8080"
+		log.Printf("Defaulting to port %s", port)
+	}
+
+	log.Printf("Listening on port %s", port)
+	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%s", port), nil))
+}
+
+func certs() (map[string]string, error) {
+	const url = "https://www.gstatic.com/iap/verify/public_key"
+
+	if len(cachedCertificates) != 0 { // Already got them previously
+		return cachedCertificates, nil
+	}
+
+	resp, err := http.Get(url)
+	if err != nil {
+		return cachedCertificates, err
```

Comment:
Please add context to the error values. https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#return-errors

```suggestion
		return nil, fmt.Errorf("http.Get: %v", err)
```